### DISCUSSION TITLE
Remove command

### DISCUSSION THREAD

**Discussion by lexesv** [emersion/maddy#429] [Q&A] [answered]
This is how messages are deleted one at a time.
`maddyctl  imap-msgs remove support@domain INBOX 1`

But how do you clear the entire folder, for example?

**Comment by lexesv** [comment, marked as answer]
`maddyctl imap-msgs remove support@domain.com INBOX 1:10000  `
Right?

**Reply by foxcpp** [reply]
This is IMAP seqset syntax, so you can use `1:*` to mean "all messages in the folder"

**Comment by lexesv** [comment]
Thank you very much.
